core/src/common/lwm2m_ipc.c

```diff
@@ -63,11 +114,11 @@ TreeNode IPC_AddClientNode(TreeNode clientsNode, const char * clientID)
     return clientNode;
 }
 
-int IPC_SendResponse(TreeNode responseNode, int sockfd, struct sockaddr * fromAddr, int addrLen)
+int IPC_SendResponse(TreeNode responseNode, int sockfd, const struct sockaddr * fromAddr, int addrLen)
 {
     int rc = 0;
     // Serialise response
-    char buffer[MAXBUFLEN] = { 0 };
+    char buffer[IPC_MAX_BUFFER_LEN] = { 0 };
```

Comment:
I know this isn't new, but is it sensible to allocate such a large buffer on the stack?